### ISSUE TITLE
Substeps support for API testing

### ISSUE THREAD

**Issue by avs01** [Substeps/substeps-framework#22]
Is there any support / documentation for producing auto tests that make requests directly to an API, as oppose to being reliant on using a web interface?

**Comment by iantmoore** [comment]
No, not at present I'm afraid, although I can see it would be useful... maybe in the future..

@beercan1989 created this library: https://github.com/beercan1989/substeps-restdriver which might do something of what you're after... you'd have to ask James about docs / capabilities / features etc.
